.github/actions/deploy/deploy.py

```
@@ -0,0 +1,1246 @@
+        if result.returncode != 0:
+            print(
+                f'❌ Deployment {deployment_name} was not created within {deployment_wait_time} minute timeout'
+            )
+            print(f'Investigating Deployment: {deployment_name}')
+            self._investigate_deployment_failure(self.deployment_namespace,
+                                                 deployment_name)
+            print(f'Investigating Deployment: {self.operator_deployment}')
+            self._investigate_deployment_failure(self.operator_namespace,
+                                                 self.operator_deployment)
+            raise RuntimeError(
+                f'Deployment {deployment_name} was not created within timeout')
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)